.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+### Subject of the issue/enhancement/features
```

Comment:
This file may need to be different for CLI. I'm not sure the standard FW/plugins one is entirely suitable